app/src/main/java/com/activities/MainActivity.java

```diff
@@ -137,13 +126,20 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 startActivity(i);
                 break;
             case R.id.sampleDatabase:
-                userDatabase.loadSampleDatabase();
+                // TODO: remove after all basic workflow is finished
+                modelObject.userDatabase.loadSampleDatabase(getApplicationContext());
                 break;
             case R.id.loadDatabase:
-                userDatabase.loadDatabase();
+                // TODO: remove after all basic workflow is finished
+                modelObject.userDatabase.loadDatabase();
                 break;
             case R.id.saveDatabase:
-                userDatabase.saveDatabase();
+                // TODO: remove after all basic workflow is finished
```

Comment:
maybe instead of coping same comment 4 times place it on top of this function

Reply:
I believe that the whole function will remain after finishing the workflow. Commented options will become undefined and  not recommended for use (as they will work for only one database/model and will touch functionalities which are generaly not supposed to use without general context). I commented them to highlight that there will be a point, when they will become dangerous to use.